compiler/luci/service/include/luci/Service/CircleShapeInference.h

```diff
@@ -58,11 +58,11 @@ class Algorithm final : public luci::CircleNodeVisitor<loco::TensorShape>
   // loco::TensorShape visit(const luci::CircleCeil *node) final;
   loco::TensorShape visit(const luci::CircleConcatenation *node) final;
   // loco::TensorShape visit(const luci::CircleConst *node) final;
-  // loco::TensorShape visit(const luci::CircleConv2D *node) final;
+  loco::TensorShape visit(const luci::CircleConv2D *node) final;
   // loco::TensorShape visit(const luci::CircleCos *node) final;
   // loco::TensorShape visit(const luci::CircleCustom *node) final;
   // loco::TensorShape visit(const luci::CircleDepthToSpace *node) final;
-  // loco::TensorShape visit(const luci::CircleDepthwiseConv2D *node) final;
+  loco::TensorShape visit(const luci::CircleDepthwiseConv2D *node) final;
```

Comment:
Is there a reason - that you decided to migrate both `Conv2D` and `DepthwiseConv2D` simultaneously?

Reply:
https://github.com/Samsung/ONE/pull/14125#issuecomment-2381654810 I checked it :smile: